src/pages/SharedPage.tsx

```diff
@@ -0,0 +1,29 @@
+import "../colors.css";
+import "../global.css";
+import React from "react";
+import { Layout } from "../components/common/Layout/Layout.tsx";
+import { useGetUser } from "../hooks/useGetUser.tsx";
+import Shared from "../components/Shared/Shared.tsx";
+
+interface Profile {
+  email: string;
+  profileImageSource: string;
+}
+
+function SharedPage() {
+  const { data } = useGetUser();
+  const { email, profileImageSource } = data || {};
+  const profile: Profile = data
+    ? { email, profileImageSource }
+    : { email: "", profileImageSource: "" };
```

Comment:
기본값을 미리 할당해 주시면 삼항연산자를 사용하지 않아도 될 것 같습니다.
```suggestion
  const { data } = useGetUser();
  const { email = '', profileImageSource = '' } = data || {};
  const profile: Profile = { email, profileImageSource };
```